dealership/vehicles.py

```diff
@@ -1,15 +1,39 @@
 class Vehicle(object):
-    def __init__(self, maker, model, year, base_price, miles):
-        pass
+    def __init__(self, maker, model, year, base_price, miles, multiplier):
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price= base_price
+        self.miles = miles
+        self.multiplier = multiplier
 
+    def sale_price(self):
+        return self.base_price * self.multiplier['sale']
 
-class Car(Vehicle):
-    pass
+    def purchase_price(self):
+        return self.sale_price() - (self.multiplier['purchase'] * self.miles)
 
+class Car(Vehicle):
+    def __init__(self, maker, model, year, base_price, miles):
+        car_multiplier = {
+            'sale': 1.2 ,
+            'purchase': 0.004
+        }
```

Comment:
Instead of a dictionary these seem like they would be suited to individual attributes, either of the object(`self`) or the class.